docs/content/config/advanced/kubernetes.md

```diff
@@ -460,79 +444,19 @@ Then, configure both [Postfix][docs-postfix] and [Dovecot][docs-dovecot] to expe
                   readOnly: true
     ```
 
-**Downsides**
-
-- Not possible to access mailserver via inner cluster Kubernetes DNS, as PROXY protocol is required for incoming connections.
-
-## Let's Encrypt Certificates
-
-[Kube-Lego][kube-lego] may be used for a role of Let's Encrypt client. It works with Kubernetes [Ingress Resources][k8s-network-ingress] and automatically issues/manages certificates/keys for exposed services via Ingresses.
-
-!!! example
-
-    ```yaml
-    kind: Ingress
-    apiVersion: extensions/v1beta1
-    metadata:
-      name: mailserver
-      labels:
-        app: mailserver
-      annotations:
-        kubernetes.io/tls-acme: 'true'
-    spec:
-      rules:
-        - host: example.com
-          http:
-            paths:
-              - path: /
-                backend:
-                  serviceName: default-backend
-                  servicePort: 80
-      tls:
-        - secretName: mailserver.tls
-          hosts:
-            - example.com
-    ```
-
-Now, you can use Let's Encrypt cert and key from `mailserver.tls` [Secret][k8s-config-secret] in your [Pod][k8s-workload-pod] spec:
-
-!!! example
+With this approach,
 
-    ```yaml
-    # ...
-    env:
-      - name: SSL_TYPE
-        value: 'manual'
-      - name: SSL_CERT_PATH
-        value: '/etc/ssl/mailserver/tls.crt'
-      - name: SSL_KEY_PATH
-        value: '/etc/ssl/mailserver/tls.key'
-    # ...
-    volumeMounts:
-      - name: tls
-        mountPath: /etc/ssl/mailserver
-        readOnly: true
-    # ...
-    volumes:
-      - name: tls
-        secret:
-          secretName: mailserver.tls
-    ```
+- it is not possible to access the mailserver via cluster-DNS, as the PROXY protocol is required for incoming connections.
 
+[Kustomize]: https://kustomize.io/
+[TLS]: ../security/ssl.md
```

Comment:
Convention for doc links is using the `docs-` prefix. I often keep it all lowercase, the idea with the prefix is that it'd be easy to parse docs with tooling or CLI utilities for use cases like identifying/checking internal doc links :man_shrugging: 

Maybe it's over thinking, and isn't necessary.

I wanted to add it to our CI at one point so our docs wouldn't miss changes breaking links. The Mkdocs build --strict feature generally works, but I do recall it having some gotchas such as not catching invalid anchor (`#)` links to jump to specific sections of our docs.

Reply:
Done with af58f352188a16c0e12bab0ec033e7eb56838531 :)